### ISSUE TITLE
Add command to generate github actions

### ISSUE THREAD

**Issue by Fuco1** [emacs-eask/cli#141]
I think a really cool feature would be to generate the github actions test yaml.  I never write it from scratch but copy-paste it from some other project, so if I could just `eask generate github-workflow` or something like that, it would be superb.

**Comment by jcs090218** [comment]
You should try `eask create`! Now it supports two things, see https://emacs-eask.github.io/Getting-Started/Commands-and-options/#-creating.

- `eask create package`
- `eask create elpa`

But these will generate the "whole" project and not a single file. And this (`generate github-worflow`) is indeed a great idea! 😁

**Comment by Fuco1** [comment]
Oh yea, the create templates are awesome! But I have many projects I'm porting from travis ci / Cask and there you want to just generate one or two files.

1. Migrate `Cask` to `Eask`
2. Add github workflow

these are the two actions I do most often.

**Comment by jcs090218** [comment]
I would like to support all of them. :)

I don't know if I should create another subcommand `eask generate` or make them all live under `eask create`. 🤔

**Comment by Fuco1** [comment]
For the cask to eask maybe that could be a flag on `eask init --from-cask`.

For the random files, I think `eask generate` is more discoverable than `eask create`.  I see `eask create` as an analogue to "create react app" where it bootstraps the whole thing.

One tool in the JS world which is really cool and has all these concepts of generators, executors etc is https://nx.dev/getting-started/intro you should check it out, it's great for inspiration.  I'm designing all our internal work tools after their philosophy.

**Comment by Fuco1** [comment]
On the other hand, there is a lot of eask commands now.  Maybe some of them could be stuffed under different top-level commands.

For example `pkg-file` could go under `generate` as well.

**Comment by jcs090218** [comment]
Some changes I've made, 

1. Move command `pkg-file` and `autoloads` under generate subcommand (#142)
2. Introduce new option `--from`. You can pass in `cask` to generate an Eask-file from a Cask-file. (#145)
3. (WIP) New command `eask generate workflow github` generates the workflow file `test.yml` (#146)

**Comment by jcs090218** [comment]
I've opened a PR for this, here is the result.

```
init [files..]

Initialize project to use Eask

Command Options:
      --from   build from an existing package                                                                                                                                                            [string]
      --files  files to use with `from` flag                                                                                                                                                              [array]
Proxy Options:
      --proxy        update proxy for HTTP and HTTPS to host                                                                                                                                             [string]
      --http-proxy   update proxy for HTTP to host                                                                                                                                                       [string]
      --https-proxy  update proxy for HTTPS to host                                                                                                                                                      [string]
      --no-proxy     set no-proxy to host                                                                                                                                                                [string]
Options:
      --version             show version number                                                                                                                                                         [boolean]
      --help                show usage instructions                                                                                                                                                     [boolean]
  -g, --global              change default workspace to ~/.emacs.d/                                                                                                                                     [boolean]
  -a, --all                 enable all flag                                                                                                                                                             [boolean]
  -q, --quick               start cleanly without loading the configuration files                                                                                                                       [boolean]
  -f, --force               enable force flag                                                                                                                                                           [boolean]
      --development, --dev  turn on development mode                                                                                                                                                    [boolean]
      --debug               turn on debug mode                                                                                                                                                          [boolean]
      --strict              report error instead of warnings                                                                                                                                            [boolean]
      --allow-error         continue the executioon even there is error reported                                                                                                                        [boolean]
      --insecure            allow insecure connection                                                                                                                                                   [boolean]
      --timestamps          log with timestamps                                                                                                                                                         [boolean]
      --log-level           log with level                                                                                                                                                              [boolean]
      --log-file, --lf      generate log files                                                                                                                                                          [boolean]
      --elapsed-time, --et  show elapsed time between each operation                                                                                                                                    [boolean]
      --no-color            disable color output                                                                                                                                                        [boolean]
  -v, --verbose             set verbosity from 0 to 4                                                                                                                                                    [number]
```

It's much nicer! :D

**Comment by jcs090218** [comment]
BTW, the two new commands are still new, so I'm looking forward to improve it! :)

- `eask init --from=cask`
- `eask generate workflow github`

**Comment by jcs090218** [comment]
This is implemented in #147; therefore, close this issue.

Feel free to open another issue(s) to improve these new commands! :D
